### PR TITLE
Integrate pallet-fixed-validators-set

### PR DESCRIPTION
**Pull request by MOZGIII**

### CODE REVIEW

File: crates/humanode-runtime/src/lib.rs [hunk 1]
```diff
@@ -563,11 +563,21 @@ impl pallet_bootnodes::Config for Runtime {
     type MaxBootnodes = ConstU32<16>;
 }
 
+impl pallet_fixed_validators_set::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type ValidatorId = AccountId;
+    type MaxValidators = ConstU32<MAX_AUTHENTICATIONS>; /* same amount in theory */
```
Comment (dmitrylavrenov) [inline crates/humanode-runtime/src/lib.rs:569]: Should we add `MaxBootnodes` number as well?

File: crates/pallet-humanode-offences/src/mock.rs [hunk 2]
```diff
@@ -35,6 +35,7 @@ frame_support::construct_runtime!(
         System: frame_system,
         Bootnodes: pallet_bootnodes,
         Bioauth: pallet_bioauth,
+        FixedValidatorsSet: pallet_fixed_validators_set,
```
Comment (dmitrylavrenov) [inline crates/pallet-humanode-offences/src/mock.rs:38]: What about adding tests to verify slashing logic for identification related to `FixedValidatorSet`?

File: crates/humanode-runtime/src/lib.rs (hunk 1 above)
Comment (MOZGIII) [reply crates/humanode-runtime/src/lib.rs:569]: Yeah, this fixed set doesn't replace the bootnodes - it is an addition rather

File: crates/pallet-humanode-offences/src/mock.rs (hunk 2 above)
Comment (MOZGIII) [reply crates/pallet-humanode-offences/src/mock.rs:38]: Right, there should be no slashing; there aren't any tests that I could modify/extend to add this, are there?

**Comment by MOZGIII** [comment]
I have updated the PR description with some notes on the intent for this change.

Also, I'm tempted to postpone the complete testing coverage of this feature for a less urgent moment; please give me your opinion on what it critical to test in this high-urgency timing, and what we could test later.

I, for one, think we should add tests to see to it that the `pallet-session` handles the potential duplicates coming from the `pallet-humanode-session` correctly. Where should we put it? At the runtime level or at the `pallet-humanode-session`? I think `pallet-humanode-session` is a good fit for this.

File: crates/humanode-runtime/src/lib.rs [hunk 3]
```diff
@@ -563,11 +563,21 @@ impl pallet_bootnodes::Config for Runtime {
     type MaxBootnodes = ConstU32<16>;
 }
 
+impl pallet_fixed_validators_set::Config for Runtime {
```
[the remark this answers is not in the document]
Comment (dmitrylavrenov) [reply crates/humanode-runtime/src/lib.rs:566]: Done with https://github.com/humanode-network/humanode/pull/1620

[the remark this answers is not in the document]
Comment (MOZGIII) [reply crates/humanode-runtime/src/lib.rs:566]: [cc5f24c](https://github.com/humanode-network/humanode/pull/1618/commits/cc5f24c96c87d13bba17e783f54f90adf7539851)

**Comment by MOZGIII** [comment]
I've added the test I wanted and it turns out there is an issue there!

I've decided to provide a separate PR for the fix: https://github.com/humanode-network/humanode/pull/1621
Please take a look

File: crates/pallet-humanode-session/src/mock.rs [hunk 4]
```diff
@@ -34,6 +34,7 @@ frame_support::construct_runtime!(
         System: frame_system,
         Bootnodes: pallet_bootnodes,
         Bioauth: pallet_bioauth,
+        FixedValidatorsSet: pallet_fixed_validators_set,
```
[the remark this answers is not in the document]
Comment (MOZGIII) [reply crates/pallet-humanode-session/src/mock.rs]: This has been addressed
